src/commonMain/kotlin/com/github/ajalt/colormath/XYZ.kt

```diff
@@ -43,20 +43,40 @@ data class XYZ(val x: Double, val y: Double, val z: Double, val a: Float = 1f) :
     }
 
     override fun toLAB(): LAB {
+        // Equations from http://www.brucelindbloom.com/index.html?Eqn_XYZ_to_Lab.html
         fun f(t: Double) = when {
-            t > 0.008856 -> t.pow(1.0/3)
-            else -> (t * 7.787037) + (4 / 29.0)
+            t > CIE_E -> t.pow(1.0 / 3)
+            else -> (t * CIE_K + 16) / 116
         }
 
-
-        val fx = f(x / 95.047)
-        val fy = f(y / 100.0)
-        val fz = f(z / 108.883)
+        val fx = f(x / D65.x)
+        val fy = f(y / D65.y)
+        val fz = f(z / D65.z)
 
         val l = (116 * fy) - 16
         val a = 500 * (fx - fy)
         val b = 200 * (fy - fz)
 
         return LAB(l.coerceIn(0.0, 100.0), a, b, alpha)
     }
+
+    override fun toLUV(): LUV {
+        // Equations from http://www.brucelindbloom.com/index.html?Eqn_XYZ_to_Lab.html
+        fun fuv(x: Double, y: Double, z: Double) = (x + 15 * y + 3 * z).let {
+            Pair((4 * x) / it, (9 * y) / it)
+        }
+
+        val yr = y / D65.y
+        val (`u'`, `v'`) = fuv(x, y, z)
```

Comment:
Lets rename these to `uPrime`, `vPrime` etc., since the backticks combined with `'` are a bit hard to read,

Reply:
Do you think it's best to inline the function `fuv` here as well, like in `LUV.kt`? i.e.:
```kotlin
val yr = y / D65.y

val denominatorPrime = x + 15 * y + 3 * z
val uPrime = (4 * x) / denominatorPrime
val vPrime = (9 * y) / denominatorPrime

val denominatorrPrime = (x / D65.x) + 15 * yr + 3 * (z / D65.z)
val urPrime = (4 * x) / denominatorrPrime
val vrPrime = (9 * y) / denominatorrPrime

// ...
```

Reply:
Yes, that seems like a good idea.